Dockerfile

```diff
@@ -14,7 +14,7 @@ RUN \
  rm -rf /var/cache/yum/*
 
 RUN \
- yum -y install --enablerepo=osg-upcoming-rolling osg-flock \
+ yum -y install --enablerepo=osg-upcoming-testing osg-flock \
```

Comment:
I'd only note here that the other changes in this vein changed the repos from development to testing; that is, increasing the vetting level; while here it goes from rolling to testing, which decreases the vetting level.

Reply:
Yup, that's fine since this only results in `fresh` image tags and we have an additional vetting step before promoting a given `fresh` tag to stable